supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc

```diff
@@ -479,6 +479,17 @@
 
 *See also*:
 
+[discrete]
+[[codebase]]
+==== image:images/yes.png[yes] codebase (noun)
+*Description*: A _codebase_ is a complete collection of source code for a software component, application, or system. Write as shown: two words.
```

Comment:
The last sentence is contradictory. We are recommending one word.

Reply:
Thanks, @mportman12  -- I had actually spotted that error myself but didn't get a chance to add a comment. Fixed it now anyway -- ready for review again :)